Vic2ToHoI4/Source/V2World/Technology/InventionsFactory.cpp

```diff
@@ -0,0 +1,74 @@
+#include "InventionsFactory.h"
+#include "OSCompatibilityLayer.h"
+#include "ParserHelpers.h"
+
+
+
+Vic2::Inventions::Factory::Factory()
+{
+	// invention names have the usual alphanumerics, plus ����:&
+	registerRegex("[a-zA-Z0-9_.\xe8\xe9\xf6\xfc\\:\\&]+",
+		 [this](const std::string& inventionName, std::istream& theStream) {
+			 inventions->inventionNames.push_back(inventionName);
+			 commonItems::ignoreItem(inventionName, theStream);
+		 });
+}
+
+
+std::unique_ptr<Vic2::Inventions> Vic2::Inventions::Factory::getInventions(const Configuration& theConfiguration)
```

Comment:
I always associate "get" in function names with "grab from memory", certaintly not read from disk (which would be "load", from "load from tape"), but that's individual.